velox/exec/fuzzer/AggregationFuzzer.cpp

```diff
@@ -431,14 +431,20 @@ void AggregationFuzzer::go() {
 
         // 10% of times use global aggregation (no grouping keys).
         std::vector<std::string> groupingKeys;
+        bool genEmptyTable = false;
         if (vectorFuzzer_.coinToss(0.1)) {
           ++stats_.numGlobal;
+
+          // 30% of times test global aggregation with empty input.
+          genEmptyTable = vectorFuzzer_.coinToss(0.3);
```

Comment:
I noticed that we only test with empty table for global aggregation. Should we test it for non-global aggregation as well?

Reply:
Updated, thanks for the suggestion.